cram/rANS_static.c

```diff
@@ -356,7 +356,7 @@ unsigned char *rans_uncompress_O0(unsigned char *in, unsigned int in_size,
 
 unsigned char *rans_compress_O1(unsigned char *in, unsigned int in_size,
 				unsigned int *out_size) {
-    unsigned char *out_buf = malloc(1.05*in_size + 257*257*3 + 9);
+    unsigned char *out_buf;
     unsigned char *cp = out_buf, *out_end;
```

Comment:
Doesn't this move of initialisation make cp initialised from uninit'd data?  clang is complaining about it?